src/scenes/LipsyncScene.ts

```diff
@@ -0,0 +1,106 @@
+        prerequisites:
+            - src/config/captions.txt, which is a tsv of [filename \t captiontext] (or whatever file is in rhubarbconf.audioListFile location)
+            - audio files in the  rhubarbconf.audioDirectory (same audio for captions)
+         run the command below to generate lipsync.json: (the command also compresses and copies result to config.ts)
+        > npm run rhubarb => makes/updates/appends lipsync.json file in rhubarbconf.outputDirectory
+
+        use config.lipsync and config.captions in your code. See this file for examples.
+
+        note that running the rhubarb will replace existing entries and add new entries (it will not delete existing entries, same for the captions). If you want to delete the contents of lipsync.json then do it manually.
+    */
+
+    private mouthpuppet:Puppet.PuppetMouth;
+
+    private lipsync = CONFIG.lipsync as Puppet.RhubarbConfig;
+
+    preload():AssetList{
+        return [
+            {type:'animate', id:'Art', stage:Art.stage, cacheInstance:true},
+            {type:'sound',id:'hello',path:'sounds/vo/hello.{ogg,mp3}',context:'vo'},
+            {type:'sound',id:'bye',path:'sounds/vo/bye.{ogg,mp3}',context:'vo'}
+        ];
+    }
+
+    setup(){
+        this.art = this.cache.animations.Art as Art;
+        this.addChild(this.art);
+        this.mouthpuppet = new Puppet.PuppetMouth(this.art.mouth);
+        this.mouthpuppet.rest();
+    }
+
+    start(){
+
+        this.art.playBtn.buttonMode = true;
+        this.art.playBtn.interactive = true;
+        this.art.playBtn.on('pointerup', (ev:PIXI.interaction.InteractionEvent) => {
+            this.stageManager.showCaption('hello');
+            this.playPuppetExample(this.mouthpuppet, "hello");
+        } );
+
+        this.art.stopBtn.buttonMode = true;
+        this.art.stopBtn.interactive = true;
+        this.art.stopBtn.on('pointerup', (ev:PIXI.interaction.InteractionEvent) => {
+            console.log("stop the sound, lipsync, and captions");           
+            this.sound.vo.stopAll();
+            this.mouthpuppet.stop();
+            this.stageManager.stopCaption();
+        } );
+
+        this.art.homeBtn.buttonMode = true;
+        this.art.homeBtn.interactive = true;
+        this.art.homeBtn.on('pointerup', (ev:PIXI.interaction.InteractionEvent) => {            
+            this.changeScene('game');
+        } );
+
+    }
+
+    playPuppetExample = (mouthpuppet:Puppet.PuppetMouth, soundname: string) => {
+
+        if(this.lipsync[soundname]) {
+            let soundinstance = this.sound.play(soundname) as PIXI.sound.IMediaInstance;
+            mouthpuppet.lipSync(soundinstance, this.lipsync[soundname]);
+        } else {
+            console.warn("No lipsync for ", soundname);
+        }
+    }
+
+    cleanup(){
+        
+        this.sound.vo.stopAll();
```

Comment:
maybe instead of putting this in the cleanup (at the midpoint of the transition animation), we should stop the sound when we call `changeScene()`. Should also stop the caption and puppet mouth, maybe take the callback for the stop button and make it its own named function so that it can be called in multiple places. 

Separately, I think we should avoid using stopAll(), as it's rather inefficient, calling stop() on every loaded sound whether it's playing or not - better to keep track of what we're playing when possible.

Reply:
good feedback, I made those changes